src/lib/caller.unit-test.ts

```diff
@@ -30,6 +30,6 @@ let tx = await Mina.transaction(privateKey, () => {
 });
 
 // according to this test, the child doesn't get token permissions
-expect(tx.send()).rejects.toThrow('Token_owner_not_caller');
-
-shutdown();
+await expect(tx.send()).rejects.toThrow(
+  'can not use or pass on token permissions'
+);
```

Comment:
these tests were actually not working 🤦🏻‍♂️ because `expect.rejects.toThrow` is an async function, and wasn't awaited before calling shutdown, it never finished